arrivals-example/src/main/scala/com/pagerduty/arrivals/example/ExampleAggregator.scala

```diff
@@ -0,0 +1,27 @@
+package com.pagerduty.arrivals.example
+
+import akka.NotUsed
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
+import com.pagerduty.arrivals.impl.aggregator.OneStepAggregator
+
+object ExampleAggregator extends OneStepAggregator[Int, String, String] {
```

Comment:
I think the example would read a little clearer if we use a type alias or case class for the type parameters here.

Reply:
Yeah, you're right it's not great. What I would really like is [named type arguments](https://dotty.epfl.ch/docs/reference/named-typeargs.html), but we'll have to wait a little for that.

I'm a little unclear on what you mean by using a case class, but I can certainly add type aliases.